packages/plugin-ext/src/main/browser/style/tree.css

```diff
@@ -38,3 +38,10 @@
     align-items: center;
     height: 100%;
 }
+
+.theia-tree-view  .theia-TreeContainer .theia-TreeViewInfo {
+    color: var(--theia-list-inactiveSelectionForeground);
```

Comment:
i don't think you need to declare it at all, it will inherit proper foreground

Reply:
@vrubezhny Could you remove this color please after that I think we should merge it?
@tsmaeder would it be fine with you or you want to have a look again?

Reply:
Yes, now it works. However, previously a `gray`-like color was selected as the foreground without this definition.